app/services/recurring_contribution/subscriptions/processor.rb

```diff
@@ -54,10 +54,17 @@ def paid_with_credit_card?
 
   def assign_attributes(with_save, attributes)
     subscription.assign_attributes(attributes)
+    update_user_cpf
 
     subscription.save if with_save && !has_errors?
   end
 
+  def update_user_cpf
+    return register_error(:donator_cpf, :blank) if subscription.donator_cpf.blank?
```

Comment:
What do you think of addind the field as required on the model and do something like `subscription.valid?`, I think this attr checking is more a model responsibility than the service, just a thought...